modules/plugin/property/src/main/java/org/geotools/data/property/PropertyDataStore.java

```diff
@@ -147,4 +147,21 @@ protected ContentFeatureSource createFeatureSource(ContentEntry entry) throws IO
             return new PropertyFeatureSource( entry, Query.ALL );
         }
     }
+
+    @Override
+    public void removeSchema(Name typeName) throws IOException {
+        super.removeSchema(typeName.getLocalPart());
```

Comment:
This will always throw an UnsupportedOperationException. Should be:
```
removeSchema(typeName.getLocalPart());
```

In order to call the other implementation.

Maybe also add a test of this method (using a `Name`) to your test case?